src/src.pro

```diff
@@ -17,7 +17,7 @@ CONFIG += pedantic
 CONFIG += debug
 
 pedantic {
-    linux-g++ | linux-g++-64 {
+    linux-*g++ | linux-g++-64 {
```

Comment:
When looking into .../qtbase/mkspecs I can see:

- linux-g++
- linux-g++-32
- linux-g++-64

As long as you do not have your own spec files the suggested wildcard does not match any of them ?

Reply:
There is also linux-oe-g++, which wasn't caught by the check but should be, I guess

Reply:
This seems to be a 3rd-party patch from the OpenEmbedded project - probably also used by Yocto ?
As far as I can see there is only a  linux-oe-g++ spec - without 32/64 variations, right ?

Not 100% sure but does the following catch your situation: linux:gcc { ... }
Might be possible to drop the linux scope to also catch mingw builds on Windows.

But in the end the pedantic flag is intended for development only and as the available extra checks heavily depend on the specific gcc version it might be better to disable the pedantic flag as default ?

Reply:
Right, I'm building with Yocto. Using linux:gcc also works fine in my case and might be the better check in general. As for using pedantic: I tend to having it enabled too so I don't mind leaving it as it is.

Reply:
Using "gcc" only now to have the pedantic checks on other operating systems using gcc too. However I disabled debug/pedantic as default setting as those can always be enabled from the command line .